src/Components/Endpoints/test/Binding/FormDataMapperTests.cs

```diff
@@ -10,6 +10,7 @@
 using System.Globalization;
 using System.Runtime.Serialization;
 using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json.Linq;
```

Comment:
```suggestion
```
😲

Reply:
I hate the auto add namespace from VS precisely for this.